user/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+
+class IsAdminOrIfAuthenticatedReadOnly(BasePermission):
+    def has_permission(self, request, view):
+        return bool(
+            (
+                request.method in SAFE_METHODS
+                and request.user
+                and request.user.is_authenticated
+            )
+            or (request.user and request.user.is_staff)
+        )
```

Comment:
permissions should be in cinema/

Reply:
Ok, but I thought that permissions related with user should be in /user